test-cases/performance/perf-search-best-throughput-config.yaml

```diff
@@ -0,0 +1,32 @@
+test_duration: 3000
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=50 -pop seq=1..30000000"
+prepare_write_cmd: "cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000"
+stress_cmd_r: "cassandra-stress read no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=50 -pop 'dist=gauss(1..30000000,15000000,1500000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=50 -pop 'dist=gauss(1..30000000,15000000,1500000)' "
+
+n_db_nodes: 3
+n_loaders: 2
+n_monitor_nodes: 1
+
+num_loaders_step: 0
+stress_threads_start_num: 50
+add_num_threads_step: 25
+stress_step_duration: '15m'
+max_deviation: 5 # 5% relative difference
+n_stress_process: 1
+
+instance_type_db: 'i2.2xlarge'
+instance_type_loader: 'c4.2xlarge'
+instance_type_monitor: 't3.small'
+
```

Comment:
@aleksbykov let's mention here the loader AMI (even if it's the same as the defaults).
I want it to be clearly visible where it's taken from.
Also mention in the comments that this is a shard-aware loader.

Reply:
Done